### DISCUSSION TITLE
Dockerfile: multiple ARG in image tag

### DISCUSSION THREAD

**Discussion by woylie** [renovatebot/renovate#23439] [Suggest an Idea]
### Tell us more.

Renovate doesn't seem to pick up Docker image tags if they are put together with multiple `ARG`s. It would be great if it would do this.

For example, the default Dockerfile generated by the Phoenix framework looks like this:

```dockerfile
ARG ELIXIR_VERSION=1.15.3
ARG OTP_VERSION=26.0.2
ARG DEBIAN_VERSION=bullseye-20230612-slim

ARG BUILDER_IMAGE="hexpm/elixir:${ELIXIR_VERSION}-erlang-${OTP_VERSION}-debian-${DEBIAN_VERSION}"
ARG RUNNER_IMAGE="debian:${DEBIAN_VERSION}"

FROM ${BUILDER_IMAGE} as builder

# ...

FROM ${RUNNER_IMAGE}

# ...
```

**Comment by viceice** [comment]
not possible yet and very complicated to solve
